.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+*Replace this paragraph with a description of what this PR is changing or adding, and why.*
+
+*List which issues are fixed by this PR. You must list at least one issue.*
+
+*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*
+
+## Pre-launch Checklist
+
+- [ ] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
+- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
+- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
+- [ ] I read and followed the [Flutter Style Guide].
+- [ ] I listed at least one issue that this PR fixes in the description above.
+- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
+- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy].
+- [ ] I updated `CHANGELOG.md` to add a description of the change.
+- [ ] I updated/added relevant documentation (doc comments with `///`).
+- [ ] I signed the [CLA].
+- [ ] All existing and new tests are passing.
+
+If you need help, consider asking for advice on the `#hackers-new` channel on [Discord].
```

Comment:
Should we use 💭 instead of `new`?

Reply:
The full name is `#hackers-new-🐣`, but we might change the emoji on people and the name is what really matters.

Reply:
Oh I thought that contributors might use hackers-💭 more often right now.